### PR TITLE
unstyle stepwise lists chemistry

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #1521

### CODE REVIEW

**Comment by ajluk** [comment]
@brittany-johnson I have added you as a reviewer to make sure hiding the lists style is the right solution in this case. 
now these Examples look like this: 
image

File: styles/designs/carnival/parts/notes/_notes-components.scss [hunk 1]
```diff
@@ -284,6 +284,14 @@ $IconNoteAnswerTitleComponent: (
...
   )
 );
+$IconNoteStepwiseOrderedList: (
+  _name: "StepwiseOrderedList",
+  _subselector: " > ol.os-stepwise",
+  _properties: (
+    list-style-type: none,
+  ),
+);
+
```
Comment (wilkus27) [inline styles/designs/carnival/parts/notes/_notes-components.scss]: I'm not sure whether step lists occurs only in Icon Notes. Maybe safer solution is to add this settings to common-styles.scss file? @brittany-johnson what do you think?

Comment (brittany-johnson) [reply styles/designs/carnival/parts/notes/_notes-components.scss]: I just asked Larissa if these occur in other locations. I'll update you once I know.

Comment (brittany-johnson) [reply styles/designs/carnival/parts/notes/_notes-components.scss]: Yes, there will be other locations where stepwise steps will eventually appear, so they will not always be in notes. 
I think that idea will work. @wilkus27

Comment (ajluk) [reply styles/designs/carnival/parts/notes/_notes-components.scss]: @wilkus27 @brittany-johnson I have moved the styles for stepwise lists to common-styles.scss file and it works fine in this case:
image

**Pull request merged**